### ISSUE TITLE
KILL button not working in 0.203-e.0.7 UI

### ISSUE THREAD

**Issue by msdotnetclr** [starburstdata/facebook-presto#54]
The KILL button in 0.203-e.0.7 UI does not seem to be working. When the button is pushed on a query details page (e.g. https://myserver:8443/ui/query.html?20180828_153406_00009_2vwx5), the following error can be seen from JavaScript console:

```
Failed to load resource: the server responded with a status of 404 (Not Found)
PUT https://myserver:8443/ui/v1/query/20180828_153406_00009_2vwx5/killed 404 (Not Found)
```

Inspect the PUT request with Google Chrome Devtools, it reveals:

```
Request URL: https://myserver:8443/ui/v1/query/20180828_153406_00009_2vwx5/killed
Request Method: PUT
Status Code: 404 Not Found
```

I think the URL path should have been (without /ui/) https://myserver:8443/v1/query/20180828_153406_00009_2vwx5/killed.

**Comment by msdotnetclr** [comment]
The following JavaScript snippet can successfully kill a running query. Note the URL does not have /ui/ after the port number. If /ui/ is added, the 404 Not Found error above will appear.

```javascript
var url = 'https://myserver:8443/v1/query/20180828_153406_00009_2vwx5/killed';
var xhr = new XMLHttpRequest();
xhr.open('PUT', url);
xhr.send();
```

**Comment by msdotnetclr** [comment]
Reference: https://github.com/prestodb/presto/pull/10799

**Comment by mattsfuller** [comment]
Thanks @msdotnetclr for reporting this.
This should be queued up in our next patch release for 203e.

**Comment by kokosing** [comment]
Working on this.

**Comment by kokosing** [comment]
Fixed in 0.203-e.0.11
